python/openvino_tokenizers/convert_tokenizer.py

```diff
@@ -135,10 +141,22 @@ def convert_tokenizer(
 
     if ov_tokenizers is None:
         raise OVTypeError(f"Tokenizer type is not supported: {type(tokenizer_object)}")
-
+    
+    def add_sinks(model):
+        # Need to add sinks for assign nodes
+        from openvino.runtime import opset12 as opset
```

Comment:
```suggestion
        from openvino import opset12 as opset
```
We also can move this import to the top.

Reply:
These changes are not relevant anymore since from now on statefulness is added in genai transformation.